packages/dapp/components/lib/hooks/use-web-3.tsx

```diff
@@ -39,7 +39,7 @@ const client = createClient(
     autoConnect: true,
     connectors: [
       new MetaMaskConnector({
-        chains: defaultChains,
+        chains: defaultChains.slice(),
```

Comment:
This does it! 

![commit](https://github.com/ubiquity/ubiquity-dollar/assets/41552663/22359ae8-0a9a-454f-a18e-2691944f98d1)

You getting null on mainnet
You getting the contracts on hardhat (anvil)

Reply:
instad of showing unsupported network for anvil

Reply:
this one, it's going to be refactored but not on this PR, on this PR i will leave as it, fetching for all the contracts

the refactor is to make sure it doesn't look for anvil on mainnet network and just for dev or debug variable is activated